plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -1251,7 +1332,7 @@ def validate_radio_series(self, ap_config):
                         invalid_entry = "Access Point series '{0}' not supported for the radio type {1} allowed series {2}".format(
```

Comment:
We're checking is_valid on line 1327 and do break. Why do we need to check is_valid again on line 1331?
Can you remove this check?

Reply:
removed if not  condition.